parity/configuration.rs

```diff
@@ -649,6 +649,8 @@ impl Configuration {
 			return Ok(GasPricerConfig::Fixed(to_u256(dec)?));
 		} else if let Some(dec) = self.args.arg_min_gas_price {
 			return Ok(GasPricerConfig::Fixed(U256::from(dec)));
+		} else if self.args.arg_chain != "foundation" {
+			return Ok(GasPricerConfig::Fixed(U256::zero()));
```

Comment:
Should be something like `if Spec_type::from_str(self.chain()).map_or(false, |s| s == SpecType::Foundation` because there are other strings that match the foundation chain.

Reply:
Might be worth to wait for #6878 (or integrate same changes) where `self.chain()` returns a `SpecType`